integration-test/common/src/test/java/org/apache/uniffle/test/CoordinatorGrpcTest.java

```diff
@@ -121,11 +122,29 @@ public void getShuffleRegisterInfoTest() {
 
   @Test
   public void getShuffleAssignmentsTest() throws Exception {
-    String appId = "getShuffleAssignmentsTest";
+    final String appId = "getShuffleAssignmentsTest";
     CoordinatorTestUtils.waitForRegister(coordinatorClient,2);
+    // When the shuffleServerHeartbeat Test is completed before the current test,
```

Comment:
https://github.com/apache/incubator-uniffle/blob/1021031b5eb1f4b97448f6431ea23e13a691b874/integration-test/common/src/test/java/org/apache/uniffle/test/CoordinatorGrpcTest.java#L249
Because this `shuffleServerConf` modifies the netty port number of a server, there will be one less machine allocated. Therefore, in order to maintain the original test, we need to set the port number back to a negative number, which is equivalent to setting the label to `GRPC`.